index.bs

```diff
@@ -24,6 +24,68 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking.  This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses a heuristic-based approach to combat navigational-tracking. Safari uses
+a client-side machine classifier to determine which sites (determined by
+eTLD+1) are participating in navigational-tracking. The main inputs to this
+classifier are "<a href="//webkit.org/blog/7675/intelligent-tracking-prevention/">
+subresource under number of unique domains, sub frame under number of unique
+domains, and number of unique domains redirected to</a>".
+
+Once a site is classified as a site with cross-site tracking capabilities, it
+can "taint" other sites it participates in redirection chains with.  For
+example, consider the case of a user clicks on a link on
+<code>start.example</code>, which redirects to <code>second.example</code>,
+which redirects to <code>third.example</code>, which redirects to
+<code>fourth.example</code>, which redirects to <code>end.example</code>. If
+Safari has classified <code>third.example</code> as having tracking
+capabilities, the above behavior may result in Safari also classifying
+<code>second.example</code> and <code>fourth.example</code> has having tracking
+capabilities.
+
+Safari deploys defenses against navigational-tracking if a navigation matches
+all of the following criteria:
+
+<ul>
+  <li>The site (determined by eTLD+1) of the user is navigating <em>from</em>
+      has been classified as having cross-site tracking capabilities</li>
+  <li>The URL the user is navigating <em>to</em> has either query parameters
+      or a URL fragment</li>
+</ul>
+
+When the above criteria holds the following defenses are deployed:
+
+<ul>
+  <li>The lifetime of client-side set cookies on the <em>destination page</em>
+      is capped at 24 hours.</li>
+  <li>If the user does interact (i.e., register a user activation) on the
+      site within seven days, all non-cookie storage is data is deleted.</li>
```

Comment:
"… days of browser use …"

Reply:
We have also implemented the SameSite=strict jail mitigation which we only subject known bad actors to, currently zero. See "SameSite=Strict Cookie Jail for Bounce Trackers" in https://webkit.org/blog/11338/cname-cloaking-and-bounce-tracking-defense/.

Reply:
If Safari is not currently using the "SameSite=Strict Cookie" approach against a site, I'd prefer to not include it in this document, which is intended to only describe mitigations that are actively being used. Please let me know if Safari starts using this again though, and I will gladly update the document